services/integration/src/controllers/constitution/constitution.js

```diff
@@ -1,18 +1,22 @@
-const Constitution = require('../models/constitution.model');
-const { asyncHandler } = require('../utils');
+const Constitution = require('../../models/constitution/constitution.model');
+const { asyncHandler } = require('../../utils');
 
 const create = asyncHandler(async (req, res) => {
-  res.send({ message: 'Operation not supported' });
+  const constitution = new Constitution(req.body);
+  await constitution.save();
+  res.send('Constitution Created successfully');
```

Comment:
In these kind of cases, you should response with a JSON containing a message. like { message : 'Constitution Created successfully'}